src/anm/events.js

```diff
@@ -142,6 +142,7 @@ registerEvent('X_STOP', 'stop', 'x_stop');
 registerEvent('X_ERROR', 'error', 'x_error');
 
 // * playing (player state)
+registerEvent('S_CHANGE_STATE', 'player_change_state', 'player_change_state');
```

Comment:
All player state events are marked as `S_`, I think no need in `player_` prefix, and I hope in the nearby future we'll find some more accurate naming pattern, of course.

And yeah, I forgot to register it when introduced it, thanks for finding that!